Library/Formula/mlkit.rb

```diff
@@ -0,0 +1,43 @@
+class Mlkit < Formula
+  desc "Compiler for the Standard ML programming language"
+  homepage "https://melsman.github.io/mlkit"
```

Comment:
Looks like this redirects to http://www.elsman.com/mlkit but it won't load in my browser. What's the current homepage?